ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -35,22 +62,15 @@
           auto-close-on-fail: false
 
     scm:
-      - git:
-          url: https://github.com/ceph/ceph.git
-          branches:
-            - ${sha1}
-          refspec: +refs/pull/*:refs/remotes/origin/pr/*
-          browser: auto
-          timeout: 20
-          skip-tag: true
-          wipe-workspace: true
+      - ceph-build
+      - ceph
 
     builders:
-      - shell: "ccache -M 5G; ccache -z; export NPROC=$(nproc); timeout 7200 ./run-make-check.sh; ccache -s"
+      - shell: "cd ceph; ../ceph-build/ceph-pull-requests/build/build build"
```

Comment:
You can use ``!include_raw`` here.  See: https://github.com/ceph/ceph-build/blob/master/ceph-build/config/definitions/ceph-build.yml#L56